setup.py

```diff
@@ -98,6 +98,7 @@ def load_deps(path):
     "google-cloud-storage": ["gcsfs~=2021.8.1"],
     "google-cloud-bigquery": ["google-cloud-bigquery~=3.0"],
     "kafka": ["kafka-python~=2.0"],
+    "redis": ["redis-py-cluster~=2.1.3", "redis~=3.5"],
```

Comment:
Please note that adding a new extra reqs category requires changes in the package test (see `NOTE` in this same file). Let's discuss this if you need explanation on what to do there.

Reply:
added